### ISSUE TITLE
Arguments to bind shorthand

### ISSUE THREAD

**Issue by bbrk24** [DanielXMoore/Civet#954]
`x@y z` becomes `x.y.bind(x)(z)`. I think `x.y.bind(x, z)` would be a more useful interpretation of that statement.

**Comment by STRd6** [comment]
I agree that this could be useful but it is also sometimes useful to invoke a bind immediately as well. We should see if we can discover an unambiguous syntax that would allow for both cases.

**Comment by bbrk24** [comment]
Is `x.y.bind(x)(z)` meaningfully different from `x.y(z)`?

**Comment by STRd6** [comment]
Good point, I can't think of any examples other than some esoteric cases but we probably don't want to optimize for those.
